DataDefinitions/Ship.cs

```diff
@@ -35,7 +35,7 @@ public class Ship : INotifyPropertyChanged
 
         /// <summary>the size of this ship</summary>
         [JsonIgnore]
-        public string size { get; set; }
+        public LandingPadSize size { get; set; }
```

Comment:
This presents a change to the data being presented to Cottle (from string to object). I had intentionally kept `size` as a user facing value for Cottle.